netdata.spec.in

```diff
@@ -1,90 +1,100 @@
+#
+# Conditional build:
+%bcond_without	systemd		# systemd
+%bcond_without	nfacct		# build with nfacct plugin
```

Comment:
yes, they are enabled everywhere, but disabled on line 8 with undefine. the condition is the same as before